docs/Installation/Installation.md

```diff
@@ -61,7 +61,7 @@ all of these dependencies.
 * BLAS (e.g. [OpenBLAS](http://www.openblas.net))
 * [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.8
 * [Boost](http://www.boost.org/) 1.60.0 or later
-* [Brigand](https://github.com/edouarda/brigand)
+* [Brigand](https://github.com/edouarda/brigand) from 4 October 2020 or later
```

Comment:
(Optional) Giving a date that isn't an official release might make it confusing which version to use. Maybe give a commit hash instead?

Reply:
Good idea.  Changed to "at commit 1c398e4f1e817ab195e4cd6fbb03c18cb386eea3 (late 2020) or later".  I decided to retain a vague date so people with a much newer download don't have to look through the repo to verify they are OK.